README.md

```diff
@@ -47,3 +47,6 @@ Configure github action at https://docs.github.com/en/pages/getting-started-with
 3. push pages to github. A build action will be triggered automatically to release the page https://github.com/hippopenny/hippopenny.github.io/actions
 
```

Comment:
can you please check in scripts to generate posts and cron job script to this repo as well?

Reply:
i will try